wordpress-mobile/WordPress-iOS
# Jetpack Section: Backup Status and Backup Complete

Part of #15191

- Hooked up download backup APIs 
    - Prepare downloadable backups
    - Get backup status (poll for status every second)
- Added a new Backup Complete screen

### Notes:
- Styling isn't 100% complete
- Buttons on Backup Complete screen are not hooked up yet
- Error handling will be addressed in a separate PR

### To test:

1. My Site > Activity Log > rewindable activity cell
2. Tap on ellipsis icon
3. Tap on `Download Backup`
4. Tap on `Create downloadable file`
     - ✅ The progress view should update (roughly 0% → 77% → 88%)
     - ✅ Once the backup is ready to download, the app should transition to the Backup Complete screen
     

https://user-images.githubusercontent.com/6711616/104443328-a639c480-55d9-11eb-8d09-d49dfdd8c102.mov



### PR submission checklist:
